mllib/src/main/scala/org/apache/spark/ml/feature/RFormula.scala

```diff
@@ -404,6 +429,30 @@ class RFormulaModel private[feature](
       s"Label column already exists and is not of type ${NumericType.simpleString}.")
   }
 
+  private def foldExprs(dataframe: DataFrame)(f: (DataFrame, String) => DataFrame): DataFrame =
+    resolvedFormula.evalExprs.foldLeft(dataframe)(f)
```

Comment:
this is very minor - perhaps save `resolvedFormula.evalExprs.foldLeft(dataframe)(f)` in a field, next to resolvedFormula which already is

Reply:
I couldn't quite understand this one. Did you mean save it as a temporary variable and return that?

Reply:
I mean instead of calling this as a method twice, save the return into a local variable and re-use it (and maybe we don't need a method then)

Reply:
Got it now, thanks.